ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp

```diff
@@ -98,15 +99,8 @@ void ViewshedCamera::calculateViewshed()
 
 void ViewshedCamera::setInitialViewpoint()
 {
-  // create a camera and set the initial viewpoint
-  const double x = -4.49492;
-  const double y = 48.3808;
-  const double z = 48.2511;
-  Point pt(x, y, z, SpatialReference(4326));
-  const double heading = 344.488;
-  const double pitch = 74.1212;
-  const double roll = 0;
-  Camera camera(pt, heading, pitch, roll);
-  Viewpoint initViewpoint(pt, camera);
-  m_scene->setInitialViewpoint(initViewpoint);
+  const Point initialPt(2.82691, 41.98500, 124.987, SpatialReference::wgs84());
```

Comment:
This is fine to change. Just to give some backstory to this - at one point we had feedback that when we pass a bunch of doubles into a c'tor or method, it was difficult to tell what each argument actually was - people would need to then go to the API ref just to see what the code was doing. We countered this by being more verbose in defining variables with good names. In some cases it ends up being a bit over verbose (as is the case here). The key is to strike a balance

Reply:
@ldanzinger That's a good point and I've reverted the style change.